app/views/projects/_import_export.html.erb

```diff
@@ -1,13 +1,22 @@
 <details id="import-export">
   <summary class="import-export-header"">
     <i class="fa fa-lg fa-chevron-down rotate"></i>
-    <h4 class="import-export-header__text">CSV Import / Export</h4> </summary>
+    <h4 class="import-export-header__text">CSV Import / Export</h4>
+  </summary>
   <section>
     <div id="import" class="card-deck mb-3">
       <div class="card mb-6 box-shadow">
         <div class="card-header">
           <h4 class="my-0 font-weight-normal">Import CSV</h4>
         </div>
+        <div id="instructions">
+          <b>Instructions:</b>
+          <ul class="list">
+            <li class="list-item">The file must have the following columns and headers: id, title (required), description and position.</li>
```

Comment:
the second header should be `title`, not `title (required)` from the code https://github.com/fastruby/points/blob/main/app/controllers/stories_controller.rb#L77